src/pretix/base/models/log.py

```diff
@@ -165,6 +165,16 @@ def parsed_data(self):
     def delete(self, using=None, keep_parents=False):
         raise TypeError("Logs cannot be deleted.")
 
+    @classmethod
+    def bulk_create_and_postprocess(cls, objects):
+        if connections['default'].features.can_return_rows_from_bulk_insert:
+            cls.objects.bulk_create(objects)
+            cls.bulk_postprocess(objects)
+        else:
+            for le in objects:
+                le.save()
+            cls.bulk_postprocess(objects)
```

Comment:
One could extract the `cls.bulk_postprocess(objects)` from the if-else to make it a tiny bit more obvious that only the create is different in bulk.

```suggestion
        else:
            for le in objects:
                le.save()
        cls.bulk_postprocess(objects)
```